src/main/java/za/co/neslotech/spring/trainspringbootinsight/controller/UserController.java

```diff
@@ -13,6 +13,7 @@ public class UserController extends CrudController<User, Long> {
     private final UserService service;
 
     public UserController(final UserService service) {
+        setReadOnly(true);
```

Comment:
How do you handle the use case where you need to update your user? Your controller should not be readonly in that case.